Common/src/main/java/com/blamejared/crafttweaker/impl/util/sequence/SequenceManager.java

```diff
@@ -0,0 +1,113 @@
+package com.blamejared.crafttweaker.impl.util.sequence;
+
+import com.blamejared.crafttweaker.api.util.sequence.Sequence;
+import com.blamejared.crafttweaker.api.util.sequence.SequenceType;
+
+import java.util.*;
+
+/**
+ * Manages {@link Sequence}s, handles adding, removing and ticking.
+ */
+public class SequenceManager {
+    
+    private static final Map<SequenceType, List<Sequence<?, ?>>> clientSequences = new HashMap<>();
+    private static final Map<SequenceType, List<Sequence<?, ?>>> clientSequencesView = Collections.unmodifiableMap(clientSequences);
+    private static final Map<SequenceType, List<Sequence<?, ?>>> serverSequences = new HashMap<>();
+    private static final Map<SequenceType, List<Sequence<?, ?>>> serverSequencesView = Collections.unmodifiableMap(serverSequences);
```

Comment:
If Fabric has an equivalent for `Side`, what about something like `Map<Side, Multimap<Type, Sequence<?, ?>>>`? Just a random thought I came up with. It's also true that there won't be more sides (right Mojang? :sweat_smile:)